CHANGES.txt

```diff
@@ -19,6 +19,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       from 3.10 was not expected by SubstTests.py and test/Subst/Syntax.py
     - EmitterProxy rich comparison set is completed (checker warning).
       Added __le__, __gt__, __ge__.
+    - gcc tool when reading throwaway input does not need to call to_str.
```

Comment:
Fix gcc/g++ tool failing when non utf-8 text is output from "gcc --version". This is run when gcc/g++ tools are initialized to determine which version of gcc is being run.

Reply:
okay, addressed that one.